add helm chart and add instructions for its usage in README

ronnyworm opened the pull request:
I used this in one of my k8s clusters and it might be useful for other people as well.

SinTan1729 commented:
Hi, thanks for the PR. I don't know much about k8s, but it looks fine. I'll wait for comments from others in case they know about k8s. Also, would you be willing to help with any issues regarding this in the future?

ronnyworm commented:
Sure, I can help if there are issues.

SinTan1729 commented:
Hi, can you add the other options that are provided in the docker compose file to this as well? One should be able to replicate an existing setup to k8s cluster if they want to.

ronnyworm commented:
For the db_url it wouldn't make sense to modify that in a k8s environment except for where it's mounted on the host. For the latter I added a new value in the values.yaml.
Changing the port does also not really make sense in k8s as every application is more isolated than in a docker environment. The svc.yml just takes the port that sts.yml exposes.

The other options I added.

SinTan1729 commented:
Looks good, thanks.

ronnyworm commented on helm-chart/templates/issuer.yml:
```diff
@@ -0,0 +1,18 @@
+apiVersion: cert-manager.io/v1
```
I can make it an optional manifest - would you like me to do that?
I would just add an option to the helm values along those lines: certmanagerEnabled: true/false

ronnyworm commented on helm-chart/templates/ingress.yml:
```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```
Same here: do you expect a helm variable to disable this?

ronnyworm commented on helm-chart/templates/pv.yml:
```diff
@@ -0,0 +1,13 @@
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: {{ .Values.persistence.hostPath.path }}
```
Which different methods do you think should be supported?

magnusja commented on helm-chart/templates/secret.yml:
```diff
@@ -0,0 +1,10 @@
+  name: secret
+type: Opaque
+data:
+  password: {{ .Values.password }}
```
Yes this is a secret you create, but then you specify the value over the Helm config. This i bad practise as I check the helm config into a git repo. Git repos should never contain secrets. So it would make much more sense to make it only configurable which secret is being uses, and not its content.

ronnyworm replied:
See my last message, also I'd like to refer to [this ](https://github.com/SinTan1729/chhoto-url/pull/48#discussion_r2087502410). If you don't want to use a secrets manager you should keep the user-supplied values file out of your repo.  Check helm documentation here https://helm.sh/docs/chart_template_guide/values_files
